### PR TITLE
idle tips video url added to default config

### PR DESCRIPTION
**Pull request by aaronshiel**

### CODE REVIEW

File: node/src/gql/types/config.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export const ConfigType = new GraphQLObjectType({
     urlClassifier: { type: GraphQLString },
     urlGraphql: { type: GraphQLString },
     urlVideo: { type: GraphQLString },
+    idleTipsVideoUrl: { type: GraphQLString },
```
Comment (beatthat) [inline node/src/gql/types/config.ts:25]: can go diff ways, but let's follow the name convention in place here so make it `urlVideoIdleTips`

**Review by beatthat — CHANGES_REQUESTED** [review]
looks good, just name changes

**Comment by aaronshiel** [comment]
alpha tested on:
(mentor-admin) 4.2.0-alpha.26
(mentor-graphql) 4.2.0-alpha.3

File: node/test/graphql/query/config.spec.ts [hunk 2]
```diff
@@ -172,6 +172,7 @@ describe('config', () => {
       urlClassifier: '/classifier/v2',
       urlGraphql: '/graphql/v2',
       urlVideo: '/video/v2',
+      urlVideoIdleTips: 'https://youtu.be/xSu1BhuFt8A',
```
Comment (beatthat) [inline node/test/graphql/query/config.spec.ts:175]: this url is the same as the default url, so it's not really testing anything. Change to like 'https://some/custom/url'

**Review by beatthat — CHANGES_REQUESTED** [review]
Looks good. There's a note about making the override url different from the default, so let's fix that.

Also, the whole way auth was done with `extensions` in every GQL response is just plain wrong. That's not the responsibility of this PR to fix it, but I do not want every spec in our test suite infected with that `extensions` garbage.

I guess easiest alt would be...

```typescript
expect(response.body?.data?.config).eql(
  {
     urlIdleVideo: 'some/url',
     // ...
   }
)
```

File: node/test/graphql/query/config.spec.ts [hunk 3]
```diff
@@ -91,23 +84,16 @@ describe('config', () => {
...
-          urlClassifier: '/classifier',
-          urlGraphql: '/graphql',
-          urlVideo: '/video',
-          styleHeaderLogo: '',
-        },
-      },
-      extensions: {
-        newToken: '',
-      },
+    expect(response.body.data.config).to.eql({
```
Comment (beatthat) [inline node/test/graphql/query/config.spec.ts:87]: make sure to use optional chaining here...if this test fails with data=null we want it to fail as an expectation mismatch and NOT a null-pointer ref that throws from the test code

**Review by beatthat — CHANGES_REQUESTED** [review]
small inline

**Review by beatthat — APPROVED** [review]
Go ahead and release this, test it locally and then deploy to v2 if it looks good

**Pull request merged**